includes/db-error.php

```diff
@@ -48,6 +48,10 @@ public function fiad_db_error_file(){
 			$php .= PHP_EOL;
 			$php .= 'header(\'Retry-After: 3600\');';
 			$php .= PHP_EOL;
+			$php .= '$http = $_SERVER[\'HTTPS\'] ? "https://" : "http://";';

```

Comment:
Why don't we use variable for this one instead of put it directly in `$php` variable? Like the variable `$content_htaccess`? By that way, we don't need to has every single quotes we have, and it's easier for other dev to adjust this one in the future.

Reply:
I can create variables for `$_SERVER` constants for future modification, but I think we still need to have variables directly in the string because of the `__DIR__` constant. If we put it in a variable, it will actually run in the `includes/db-error.php` and result in the wrong working directory. We still need to place these variables in `wp-content/db-error.php`